strr-strata-web/app/stores/strata.ts

```diff
@@ -52,21 +52,48 @@ export const useStrrStrataStore = defineStore('strr/strata', () => {
     }
   }
 
-  // TODO: move to separate strataDashboard store?
-  const registrations = ref([])
-  const applications = ref([])
-
   const loadStrataHotelList = async () => {
-    try {
-      registrations.value = await getAccountRegistrations()
-    } catch (e) {
-      logFetchError(e, 'Unable to load account registrations')
-    }
-    try {
-      applications.value = await getAccountApplications()
-    } catch (e) {
-      logFetchError(e, 'Unable to load account applications')
-    }
+    const regs = await getAccountRegistrations<StrataRegistrationResp>()
+      .catch((e) => {
+        logFetchError(e, 'Unable to load account registrations')
+        return undefined
+      })
+      .then((response) => {
+        if (response) {
+          return (response as StrataRegistrationResp[]).filter(
+            item => item.registrationType === ApplicationType.STRATA_HOTEL
+          ).map(reg => ({
+            hotelName: reg.strataHotelDetails.brand.name,
+            number: reg.registration_number,
```

Comment:
Oh wait a minute can you just double check that the 'registration_number' works via postman to grab a specific registration? We might need the 'id' of the registrationwhen they click 'View', I'm not totally sure

Reply:
Yeah 1 min

Reply:
Yeah I'll need to use id, good catch

Reply:
fixed